File: Bedu/Kotlin/Clase1-6/Test/app/src/main/java/com/charlye934/test/MainActivity.java
package com.charlye934.test;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.CheckBox;
import android.widget.RadioButton;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    double res = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }


    public void CalculaRes(View view){
        Button btnResultado;
        double val = Double.parseDouble(String.valueOf(view.getTag()));
        res = res + val;

        btnResultado = (Button)findViewById(R.id.btnCalcular);
        btnResultado.setOnClickListener(
                new View.OnClickListener() {
                    @Override
                    public void onClick(View v) {
                        Toast.makeText(getApplicationContext(),"resultado: " + res,Toast.LENGTH_SHORT).show();
                        res = 0;
                    }
                }
        );
    }
}

File: Ejercicios/Kotlin/InjeccionDependencias/DaggerLoggin/app/src/main/java/com/example/daggerlogin/MainActivity.kt
package com.example.daggerlogin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity(),View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnEntrar.setOnClickListener(this)
    }

    override fun onClick(view: View?) {
        when(view!!.id){
            R.id.btnEntrar -> {Toast.makeText(this,"Login pulsado",Toast.LENGTH_SHORT).show()}
        }
    }
}
